packages/trpc/server/routers/viewer/organizations/getMembers.handler.ts

```diff
@@ -15,6 +15,11 @@ export const getMembersHandler = async ({ input, ctx }: CreateOptions) => {
 
   if (!ctx.user.organizationId) return [];
 
+  const isOrgPrivate = ctx.user.organization.isPrivate;
+  const isOrgAdmin = ctx.user.organization.isOrgAdmin;
+
+  if (isOrgPrivate && !isOrgAdmin) return [];
+
```

Comment:
This is returning empty array even when team is not private but org is private. so event team admin cannot view all the team members

Reply:
Oh great shout

Reply:
I just checked that this is nothing to do with this PR (still happens on main) 

I will merge this and do a follow up for history sake